about/index.md

```diff
@@ -14,45 +14,31 @@ image:
+TERRA-REF will develop an integrated phenotyping system for energy Sorghum that leverages genetics and breeding, automation, remote plant sensing, genomics, and computational analytics. The complete system will include: 
+1) automated gantry field phenotyping system (LemnaTec field scanalyzer) provided by ARPA-E; 
+2) field phenotyping dervided from sensors on unmanned aerial and ground vehicles; 
+3) automated controlled-environment phenotyping of energy Sorghum; 
+4) computational solutions for phenotype selection and prediction; 
+5) genetics, genomics and bioinformatics for phenotype-to-genotype trait associations; 
+6) data processing and computational workflows for integrated phenotyping systems; and 
+7) phenotyping data and computational pipeline standards developed by a committee selected in collaboration with the TERRA program. 
 
-Phenotyping will focus on traits that drive yield gain and bioenergy potential and will be conducted under field and controlled environmental conditions using a sorghum diversity panel, biparental cross populations, and elite lines and hybrids from structured breeding populations. This reference system will be used to characterize phenotype-to-genotype associations, on a genomic scale, that will enable knowledge-driven breeding and the development of higher-yielding energy cultivars. Although the system will initially be used to improve energy sorghum, it is directly extendable to other crops of economic and energy significance. When combined with marker-assisted breeding and genome-wide selection for sorghum improvement, this system will increase the bioenergy contribution to our total energy supply and reduce greenhouse gas emissions.
+Digital traits that drive yield gain and bioenergy potential will be extracted from sensor data collected from sorghum grown under field and controlled environmental conditions. Over 750 lines from a sorghum diversity panel, biparental cross populations, and elite lines and hybrids from structured breeding populations will be evaluated. This reference system will be used to characterize phenotype-to-genotype associations, on a genomic scale, that will enable knowledge-driven breeding and the development of higher-yielding energy cultivars. Although the system will initially be used to improve energy sorghum, it is directly extendable to other crops of economic and energy significance. When combined with marker-assisted breeding and genome-wide selection for sorghum improvement, this system will increase the bioenergy contribution to our total energy supply and reduce greenhouse gas emissions.
 
 
 
-## Reference Phenotyping System Team
```

Comment:
I'd rather err on the side of including the larger team. I definitely don't want to appear to be taking credit for this work. Just facilitating. It is still an open question whether or not this will be the project home page - probably one that we should solve sooner (e.g. before the tech summit) than later.